log/logger_test.go

```diff
@@ -0,0 +1,30 @@
+package log_test
+
+import (
+	"bytes"
+	"errors"
+	"strings"
+	"testing"
+
+	"cosmossdk.io/log"
+)
+
+func TestLoggerOptionStackTrace(t *testing.T) {
+	buf := new(bytes.Buffer)
+	logger := log.NewLogger(buf, log.TraceOption(true), log.ColorOption(false))
+	logger.Error("this log should be displayed", "error", inner())
+	if strings.Count(buf.String(), "logger_test.go") != 1 {
+		t.Fatalf("stack trace not found, got: %s", buf.String())
```

Comment:
This is currently failing, It seems like `ErrorStackMarshaler` is only called in `Err()` but not when using field.

Reply:
https://github.com/rs/zerolog/pull/560